### ISSUE TITLE
Double sided tape vs Velcro

### ISSUE THREAD

**Issue by tijs** [HannahMitt/HomeMirror#10]
How about using velcro tape instead of double sided tape. Would make it even easier to take the android device off if you need to run an update or something like that: http://www.amazon.com/Velcro-Stick-Tape-Black-VEL60211/dp/B0017S83PI

**Comment by HannahMitt** [comment]
Yeah thats a good idea. I kept the instructions to things I've tried, but if anyone tries velcro and has some pictures, I'll add them to the README.
